api/swagger.yaml

```diff
@@ -5904,6 +5904,10 @@ paths:
           description: "The volume was created successfully"
           schema:
             $ref: "#/definitions/Volume"
+        304:
+          description: "The volume has already been created"
+          schema:
+            $ref: "#/definitions/ErrorResponse"
```

Comment:
Hi, @yongtang Thanks for the update.
I am afraid that I still have some confusion that whether status code 3xx is related to some kinds of error `$ref: "#/definitions/ErrorResponse"`, though I found in other places about 304 in swagger.yml, there is also has an error schema?
I once remember that some time some place you mentioned that in a network framework of programming language, 3xx means no error as well.